templates/xcode.pkr.hcl

```diff
@@ -127,7 +127,9 @@ build {
     inline = [
       "source ~/.zprofile",
       "brew install graphicsmagick",
+      "brew install imagemagick",
```

Comment:
Could you please move it a line above to group tools for image processing:

`brew install graphicsmagick imagemagick`